apps/i18n/common/en_us.json

```diff
@@ -844,7 +844,9 @@
   "libraryCreatorError": "There was an error creating your library. Contact support@code.org to resolve the issue.",
   "libraryDescriptionPlaceholder": "Write a description of your library",
   "libraryExportNoCommentError": "This function cannot be exported until you add a comment to it.",
+  "libraryExportSubtitle": "Share functions in your project to others in your class or to anyone with the project's ID. Others can import your functions into their projects by going to \"Manage Libraries\" in the gear icon at the top of the toolbox.",
   "libraryExportTitle": "Export Functions as a Library",
+  "libraryFunctionRequirements": "Select the functions you want to add to the library. To help understand what the function is for, you may only export functions that have a comment directly above them in your code.",
```

Comment:
I'm guessing these strings came from product, so feel free to ignore, but "to help understand" seems like kind of odd phrasing. Whose understanding? Is it "to help importers of your library understand" or is it "to help you understand"?

Reply:
yeah, moneppo reached out to the curriculum team about this, but i haven't heard back about it, so i'm just merging this work to unblock any upcoming work in this area